### PR TITLE
Include basic commands about how to fetch exercises

### PR DESCRIPTION
**Pull request by rchavarria**
This PR improves a little bit the documentation about how to run tests by including a couple of basic commands about how to fetch exercises as it was discussed on https://github.com/exercism/xjavascript/pull/297#issuecomment-237935748

Adds some changes related to #289

### CODE REVIEW

**Comment by rchavarria** [comment]
Let's see if #289 can be closed

File: docs/TESTS.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
-After configuring `exercism` command-line client and fetching your first JavaScript exercise (see [command-line interface overview](http://exercism.io/cli)) is time to run some tests. Move to the folder where that exercise's files are located (a path similar to `<EXERCISM_HOME_DIR>/<TRACK_ID>/<EXERCISE>`) and run the tests with the `jasmine-node` command you should have installed on *Installing JavaScript* step:
+After configuring `exercism` command-line client you can fetch the very next exercise
+
+    exercism fetch javascript
+
+or you can fetch a specific exercise, passing the exercise name after the language. For example, to fetch the `bob` exercise:
+
+    exercism fetch javascript bob
+
+No, it's time to run some tests. Move to the folder where that exercise's files are located (a path similar to `<EXERCISM_HOME_DIR>/<TRACK_ID>/<EXERCISE>`) and run the tests with the `jasmine-node` command you should have installed on *Installing JavaScript* step:
```
Comment (matthewmorgan) [inline docs/TESTS.md]: Should this begin `Now, it's time to run some tests`  and then later `...installed on the *Installing JavaScript* step:` ?

**Comment by rchavarria** [comment]
Fixed typos, rebased and squashed 🎆

**Comment by matthewmorgan** [comment]
:1st_place_medal: Thanks @rchavarria !

**Comment by rchavarria** [comment]
Wow! Lots of activity recently :rofl:

**Pull request merged**